framework/include/eventloop/eventloop.h

```diff
@@ -44,8 +44,39 @@ enum el_result_error_e {
 	EVENTLOOP_OUT_OF_MEMORY = -3,
 	EVENTLOOP_LOOP_FAIL = -4,
 	EVENTLOOP_NOT_FINISHED = -5,
+	EVENTLOOP_INVALID_HANDLE = -6,
 };
 
+/**
+ * @brief Pre defined Event list
+ * @details These values can be used for event type of event loop.\n
```

Comment:
These values can be used? or These values are used?
And let's use active voice instead of passive voice.
```
Event loop uses these values as event types.
```